django_mailbox/google_utils.py

```diff
@@ -17,37 +20,45 @@ class RefreshTokenNotFound(Exception):
 
 
 def get_google_consumer_key():
+    if SocialApp:
+        app = SocialApp.objects.filter(provider='google').first()
+        if app:
+            return app.client_id
     return settings.SOCIAL_AUTH_GOOGLE_OAUTH2_KEY
 
 
 def get_google_consumer_secret():
+    if SocialApp:
+        app = SocialApp.objects.filter(provider='google').first()
```

Comment:
Picking the first all auth google account is not deterministic enough if it's used to provide end-user authentication...